src/extension/background-script/actions/blocklist/add.ts

```diff
@@ -0,0 +1,23 @@
+import db from "../../db";
+
+const add = async (
+  message: { args: { host: string; name: string; imageURL: string } },
+  sender: unknown
```

Comment:
Not completetely sure but we use this in other places, does it work here as well?  
https://github.com/getAlby/lightning-browser-extension/blob/master/src/common/lib/utils.ts/#L137
@dylancom what do you think?

Reply:
Action/message handling has been updated. Please have a look at i.e.: https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/actions/accounts/add.ts#L6  

All `blocklist` actions should follow this example. Ideally [with Jest tests](https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/actions/accounts/__tests__/add.test.ts) as well but could be done later.

Reply:
@pseudozach can you help with this and have a look?